tests/Unit/Routes/WebRoutesTest.php

```diff
@@ -1,20 +1,153 @@
 <?php
 
-namespace Tests\Unit\Routes;
-
 use Tests\TestCase;
 use Illuminate\Foundation\Testing\DatabaseMigrations;
 
 class WebRoutesTest extends TestCase
 {
+    use DatabaseMigrations;
+
+    private $user = null;
+    private $centre = null;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->centre = factory(App\Centre::class)->create();
+
+        // Create a User
+        $this->user =  factory(App\User::class)->create([
+            "name"  => "test user",
+            "email" => "testuser@example.com",
+            "password" => bcrypt('test_user_pass'),
+            "centre_id" => $this->centre->id,
+        ]);
+    }
+
     /**
-     * Verify content on dashboard.
+     * Verify login direct.
      *
      * @return void
+     * @test
      */
-    public function testLoginRoute()
+    public function testLoginGuestRoute()
+    {
+        Auth::logout();
+        $this->get(URL::route('service.login'))
```

Comment:
I don;t think you need the URL facade.

Reply:
Without it, it was defaulting to "http://localhost/" rather than the app's dev setting.

Reply:
hmmm ok.